README.md

```diff
@@ -1,2 +1,19 @@
-# yad2_bouncer
-A script for bouncing yad2 ads
+# Yad2 Bouncer
+A script for bouncing yad2 ads.
+
+# Usage
+Run in the terminal the following line:
+```
+python3 src\main.py <chrome_drive> <username> <password>
+```
+
+# Requirements
+* Python 3
+* Chrome driver - bin\chromedriver.exe
+    + Download at https://sites.google.com/a/chromium.org/chromedriver/home.
```

Comment:
Explain how to get the driver on both Windows and Linux.
For Linux you can reference the makefile.